benchmarks/lucet-benchmarks/src/modules.rs

```diff
@@ -26,7 +27,12 @@ pub fn null_mock() -> Arc<dyn Module> {
     extern "C" fn f(_vmctx: *mut lucet_vmctx) {}
 
     MockModuleBuilder::new()
-        .with_export_func(b"f", f as *const extern "C" fn())
+        .with_export_func(
+            b"f",
+            unsafe { std::slice::from_raw_parts(f as *const extern "C" fn() as *const u8, 1) },
```

Comment:
I added a [`function_bytes_slice`](https://github.com/fastly/lucet/pull/174/files#diff-af384e39937e7bd880a7bc1d0040fcfeR2) macro to make this a little more concise, but don't really feel great about making that available outside testing contexts. Very open to suggestions on a nice way for it to be available here, but not for, say, downstream use.

Reply:
We have a bunch of things in `lucet_runtime_internals` that are not for downstream consumption outside of testing, e.g. `MockModuleBuilder`, so I don't see harm in putting it in the internals module.